test/integration/visitor_can_add_items_to_cart_test.rb

```diff
@@ -104,4 +104,64 @@ class VisitorCanAddItemsToCart < ActionDispatch::IntegrationTest
     assert page.has_content?("Super yummy")
     assert page.has_content?("Total: $20")
   end
+
+  test "user can adjust the quantity of an item in the cart" do
+    skip
+    category_1 = Oil.create(name: "Lard")
+    category_2 = Oil.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
+    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'd trade slota for these!", oil_id: category_2.id)
+    item_3 = Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_id: category_2.id)
+    item_4 = Chip.create(name: "Lard Yummies", price: 19, description: "Chock Full of Lard", oil_id: category_1.id)
+
+    visit chips_path
+
+    within("#slotachips") do
+      click_button "Add to Cart"
+    end
+
+    visit cart_chips_path
+
+    within ('#slotachips') do
+      assert page.has_content?("Slotachips")
+      within ('.quantity') do
+        assert page.has_content?("1")
+      end
+    end
+
+    click_link "+"
+
+    assert_equal cart_chips_path, current_path
+
+    within ('#slotachips') do
+      within ('.quantity') do
+        assert page.has_content?("2")
+      end
+      within ('.price') do
+        assert page.has_content?("40")
+      end
+    end
+
+    within ('.cart_total') do
```

Comment:
`(...)` interpreted as grouped expression.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.